### PR TITLE
Added the TBG (TIJK-Bot Games) responses to TIJK-Bot

### PR DESCRIPTION
**Pull request by JustIanJ**
# Summary

## Checklist

- [x] If code changes were made then they have been tested
- [ ] This PR adds a feature
  - [ ] This PR adds a slash command
  - [ ] This PR adds a view
  - [ ] This PR adds a cog
- [ ] This PR fixes an issue
- [x] This PR is **not** a code change (e.g. documentation, README, ...)

### CODE REVIEW

File: 8ball_responses.json [hunk 1]
```diff
@@ -1,14 +1,26 @@
...
-    "I am not sure about that",
-    "I do not understand",
-    "What are you going to do if I don't answer?"
-  ]
-}
+    "responses": [
+      "Yes",
+      "No",
+      "Maybe",
+      "Is my name TIJK Bot Games?",
```
Comment (codeman1o1) [inline 8ball_responses.json:6]: No, it's TIJK Bot.
Please fix this

File: 8ball_responses.json [hunk 2]
```diff
@@ -1,14 +1,26 @@
...
+      "Google it you twat",
+      "DEFINITELY",
+      "No, I am very sorry for you",
+      "I am not sure about that",
+      "I do not understand",
+      "What are you going to do if I don't answer?",
+      "I need to think about it.",
```
Comment (codeman1o1) [inline 8ball_responses.json:13]: Please remove the period at the end

File: 8ball_responses.json [hunk 3]
```diff
@@ -1,14 +1,26 @@
...
+      "Try again",
+      "1+1=3, which is true so to answer your question, yes",
+      "HELL NAH",
+      "Very doubtful",
+      "Sure",
+      "Without a doubt",
+      "Most likely",
+      "Might be possible",
+      "You'll be the judge",
+      "no... (╯°□°）╯︵ ┻━┻",
```
Comment (codeman1o1) [inline 8ball_responses.json:23]: Please fix grammar issue (capital letter at the start of a sentence)

File: 8ball_responses.json [hunk 4]
```diff
@@ -1,14 +1,26 @@
...
+      "might be yes, but who am i to judge?"
```
Comment (codeman1o1) [inline 8ball_responses.json:24]: Please fix grammar issues (capital letter at the start of a sentence, the "i" should be uppercase)

**Review by codeman1o1 — CHANGES_REQUESTED** [review]

**Comment by JustIanJ** [comment]
I also changed L24 to be more correct.

**Review by codeman1o1 — APPROVED** [review]
LGTM

**Pull request merged**